contracts/AINFTFactory.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/interfaces/IERC721.sol";
+
+import "./AINFT721.sol";
+
+contract AINFTFactory is Ownable {
```

Comment:
Factory can generate only one Contract??

Reply:
why?

Reply:
What is the purpose of this contract? Is it manage something important within this contract?

Reply:
why do we store `_clonedAINFTContract`, `_createdAINFTContract`?

Reply:
I missed the explanation for `_clonedAINFTContract`, `_createdAINFTContract` and its getters.
Actually, they don't need to be existing in this contract. They only exist for testing scripts in hardhat.
Without this, I cannot fetch the return address of `cloneERC721` and `createERC721`.
Sorry for confusion.

Reply:
what about using these:
```javascript
      const receipt = await tx.wait(1);
      const mintEvent = filter(receipt.events, (x) => x.event === "Mint"); // FIXME: not "Mint".
```

and add Event on this AINFTFactory.sol, and fire it on `cloneERC721()`, `createAINFT721()`.

Reply:
add Event would be more desirable!
I will add `CloneAINFT721` and `CreateAINFT721` in AINPayment and remove them.